schema/rdl_schema_0.1.json

```
@@ -1025,6 +1025,37 @@
 				"Judgement",
 				"Hybrid"
 			]
+		},
+		"Temporal": {
+			"title": "Temporal coverage",
+			"type": "object",
+			"description": "The temporal period that the dataset covers.",
+			"properties": {
+				"start": {
+					"title": "Start date",
+					"type": "string",
+					"description": "The date on which the period begins. This can refer to a year (YYYY), a month (YYYY-MM) or a day (YYYY-MM-DD) as appropriate for the dataset.",
+					"format": "date-time"
+				},
+				"end": {
+					"title": "End date",
+					"type": "string",
+					"description": "The date on which the period ends. This can refer to a year (YYYY), a month (YYYY-MM) or a day (YYYY-MM-DD) as appropriate for the dataset.",
+					"format": "date-time"
+				},
+				"duration": {
+					"title": "Duration",
+					"type": "string",
+					"description": "The total number of time periods covered using ISO 8601 duration format, e.g. 50 years is given as 'P50Y'.",
```

I'm not sure that 'total number of time periods' is quite the right formulation since ISO8601 duration format expresses both the number and unit of periods.

```suggestion
					"description": "The duration of the period in ISO 8601 duration format, e.g. 'P50Y' for 50 years",
```